op-e2e/actions/proofs/holocene_invalid_batch_test.go

```diff
@@ -109,11 +113,13 @@ func Test_ProgramAction_HoloceneInvalidBatch(gt *testing.T) {
 			useSpanBatch:            true,
 			breachMaxSequencerDrift: true,
 			holoceneExpectations: holoceneExpectations{
-				preHolocene: expectations{safeHead: 0, // Entire span batch invalidated.
-					logs: sequencerOnce("batch exceeded sequencer time drift without adopting next origin, and next L1 origin would have been valid"),
+				preHolocene: expectations{
+					safeHead: 0, // Entire span batch invalidated.
+					logs:     sequencerOnce("batch exceeded sequencer time drift, sequencer must adopt new L1 origin to include transactions again"),
 				},
-				holocene: expectations{safeHead: 1800, // We expect partial validity until we hit sequencer drift.
-					logs: sequencerOnce("batch exceeded sequencer time drift without adopting next origin, and next L1 origin would have been valid"),
+				holocene: expectations{
+					safeHead: 1800, // We expect partial validity until we hit sequencer drift.
+					logs:     sequencerOnce("batch exceeded sequencer time drift, sequencer must adopt new L1 origin to include transactions again"),
```

Comment:
note, the expected log here changed because with fixed force-inclusion we were now hitting a different drop reason. the overall test is still sound.